File: TMA.Web/ClientApp/src/app/modal/TaskDto.ts
export class TaskDto {
  taskId: number | null = null;
  taskTypeId: number | null  = null;
  taskStatusId: number | null = null;
  title: string | null = null;
  description: string | null = null;
  isCompleted: boolean = false;
  createdBy: string | null = null;
  modifiedBy: string | null = null;
  createdDate: Date | null=null;
  modifiedDate: Date | null = null;
}

File: TMA.Api/Model/TaskDto.cs
using System.Text.Json.Serialization;
using TMA.Api.Common;

namespace TMA.Model
{
    public class TaskDto
    {
        public int TaskId { get; set; }
        public int TaskTypeId { get; set; }
        public int TaskStatusId { get; set; }
        public string Title { get; set; }
        public string? Description { get; set; }
        public bool IsCompleted { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public string? ModifiedBy { get; set; }
        public DateTime? ModifiedDate { get; set; }
    }
}
